src/main/java/com/crud/tasks/service/DbService.java

```diff
@@ -16,5 +16,7 @@
         return repository.findAll();
     }
 
+    public Task getTask(Long taskId) {return repository.findById(taskId).orElse(null);}
```

Comment:
metoda nie powinna nigdy zawracac nulla. Lepiej wyrzuc tutaj wyjatek np
`orElseThrow(() -> new IllegalStateException("Not found"))`

Reply:
Jeszcze lepiej jak uzyjesz tutaj castomowego wyjatku
np TaskNotFoundException

Reply:
Racja, już zmieniłem :) I chyba przez przypadek dalszą lekcję zacząłem pisać w tym branchu, zapomniałem zmienić brancha